### DISCUSSION TITLE
Raiding mechanic that allows the community to moderate selling pressure

### DISCUSSION THREAD

**Discussion by SuperSynapse** [sunflower-land/sunflower-land#1124] [Ideas]
Was part of the AMA with Steve today and talked about an idea that I think could be the best solution to remove multi-accounting, bots, and allow the community playing the game to be the net benefiters (while also having fun!)

**What we want to do:**
- Protect players playing the game and investing (let them be the primary buyers/sellers).
- Not restrict players playing the game (eliminate aggressive blanket withdrawal fees/etc).
- Maintain value in the assets (reduce selling pressure from players not interested in the game).
- Free up the Dev's time from ban and ban drama. (create a self cleansing mechanic)
- Keep the game community centric (Less 'Devs do everything', more of the community protecting itself).
- Make the game fun, exciting, and give extra things to do between updates.

<hr>

**Proposal:**

- Have a system where all players (when minting their farm) start with a skill called "Reputation". This skill starts at zero.
- When a farm mints an NFT, deposit NFTs, or deposit resources of any kind they gain Reputation
- When a farm withdraws an NFT, withdraws SFL, or withdraws resources, they lose an equal amount of Reputation. This is a simple net-in to net out.

- If a farm continues to withdraw NFTs/resources (without minting/depositing new ones) their reputation will go below zero.
- Whenever a farm's reputation goes below zero it is open to raiding from other farms for a period of time until it gets raided back to a reputation of zero, or the timer resets.
- Farms with a high enough Reputation (for example above "50") are able to raid vunerable farms.

- During a raid, a dice roll is performed and multiplied by the amount of "negative reputation". Depending on the result, a portion of that farm's resources are stolen, and some of their reputation is recovered based on the resources stolen.
- All resources are subject to being raided, but there is a floor. For example a farm will never be able to be raided below "1 SFL."
- This continues based on negative reputation remaining, and if not raided back to zero, the timer (for example "3 days")

Only net negative players will ever succumb to a raid, and only after they withdraw. This allows players wanting to "cash out" to withdraw their items and leave, but players consistently creating selling pressure are quickly dis-incentivized through the mechanic of the game. The addition of reputation can be clearly communicated at time of withdrawal, and even rewarded for those above it.

Now even players that aren't caught in anti-botting/multi-accounting system will struggle to regain their $15 investment from minting/syncing a farm while making entertainment for those playing.

<hr>

One exception I would consider is that certain NFTs (such as tier 3) would grant certain allowances for withdrawal without lowering reputation.

**For example:**
- If someone owns a Golden Cauliflower, should they be able to withdraw a certain amount of cauliflower each week before losing rep?
- If someone has a Foreman Beaver should they be able to withdraw a certain amount of wood each week before losing rep?
- If someone owns a Mysterious Parsnip, should they be able to withdraw a certain amount of parsnips each week before losing rep? etc.

**Comment by SuperSynapse** [comment]
**Example of this working:**

- Player mints farm
0 Rep
 - Player harvests crops, chops wood, mines stone
0 Rep 
- Player Mints an NFT **+10 Rep value**
10 Rep 
- Player Mints another NFT  **+10 Rep value**
20 Rep 
- Player withdraws 15 SFL **-15 Rep value**
5 Rep 
- Player withdraws 1 NFT **-10 Rep value**
-5 Rep 

 
- Player raided for 15 wood **+2 Rep Value**
-3 Rep
- Player raided for 2 stone **+2 Rep value**
-1 Rep 
- Player raided for 2 radishes **+1 Rep value**
0 Rep


- Player farms
0 Rep 
- Player mints NFT **+15 Rep value**
15 Rep 

etc etc etc...

**Comment by sir-canada** [comment]
I really like this idea, and I upvoted it!

**Comment by Yougottabekiddingme123** [comment]
Totally disagree with this concept. In a game such as COC you cannot withdraw the in game currency. However, we have SFL which can easily be bought and sold for real USD. Thus, crudely put, raiding would amount to legalizing theft of other people's investments.

I have never withdrawn an nft or sfl from the game and am in support of measures to discourage multi and botting but I think this idea is unethical.

**Reply by sir-canada** [reply]
I think you might be misunderstanding how Synapse's system works? If you don't withdraw NFT or SFL, and play the game (you're minting your own) you'll never get negative rep and never get raided. Even if you did occasionally withdraw, as long as you mint a few NFTs you're immune.

**Reply by Yougottabekiddingme123** [reply]
I think you might have misunderstood my point. If a person withdraws a
number of nfts which the game doesn't look down upon then you are
legalising/allowing other players to steal their money and investments.

The point is not that you will be safe from this if you don't withdraw or withdrawn less than what you mint.
Rather the point is legalising the theft of other peoples investment seems to be OK to you.

**Comment by SteveChickenCX** [comment]
what about instead of going both ways (negative and positive) we can just reward those who do hold to their NFT and craft them, some sort of only positive meter.

everybody starts with 0, everytime you craft an NFT you win some REP points, everytime you withdraw an NFT you lose some REP points.
you can also gain rep points by spending SFL ingame like buying pickaxes or so.

the important points are:

* you can't go negative in points , no body can steal assets from your farm. 
* if you are above certain thresholds of rep you will get bonuses. 20 rep? -> 5% crops yield. 50 rep? 5% eggs yield.
the bonuses shouldn't be that big but strong enough to incentive good behaivor.


this system is less punishing, so people won't be afraid to go negative, they will just lose their "good rep" that can be gained again.

I assume that most players will have good rep while leechers will be left behind in a 0 balance.

**Reply by Yougottabekiddingme123** [reply]
Now this is something that makes sense and encourages investment in the
Game.

A big thumbs up from me!!!

**Reply by brian-man** [reply]
> can just reward those who do hold to their NFT and craft them, some sort of only positive meter.

I wish this philosophy was embraced consistently across the crypto gaming space:
_Reward players for the behavior you want rather than punish them for the behavior you don't want._

Carrot vs. stick.  As an example, instead of withdraw taxes choose lower payouts with staking/holding rewards instead.

**Reply by SuperSynapse** [reply]
So the problem here is the path to profit.

The majority of SFL accounts are multi-accounts

These SFL accounts only serve to withdraw resources, never adding to them.

The importance of this proposal is to "kink the hose" on accounts created to make it more difficult for them to only serve as net-negative to either provide more resources to someone's farm, or to create selling pressure into the ecosystem.

Buying and selling is great! Just don't only be a seller.

As soon as you bring thought into the equation, it makes multi-accounting much more difficult.
